bundler/lib/bundler/cli/gem.rb

```diff
@@ -231,6 +237,15 @@ def ask_and_set_test_framework
       test_framework
     end
 
+    def test_framework_help_info
+      if Bundler.settings["gem.test"] == false
+        "Your choice will only be applied to this gem."
+      else
+        "Future bundle gem calls will use your choice. " \
```

Comment:
```suggestion
        "Future `bundle gem` calls will use your choice. " \
```

Reply:
Is it possible to add output assertions for each of the branches to some existing tests?

Reply:
Yep, added specs for both branches above and for when `-t` isn't needed.